.github/PULL_REQUEST_TEMPLATE/release-checklist.md

```diff
@@ -12,46 +12,45 @@ assignees: ''
 ### How to Increment Versions
 
 Zebra follows [semantic versioning](https://semver.org).
+Semantic versions look like: MAJOR`.`MINOR`.`PATCH[`-`TAG`.`PRE-RELEASE]
 
-Look for the [draft `zebrad` changelog](https://github.com/ZcashFoundation/zebra/releases) for the automatic version bump.
-This version is based on [the labels on the PRs in the release](https://github.com/ZcashFoundation/zebra/blob/main/.github/release-drafter.yml).
+The [draft `zebrad` changelog](https://github.com/ZcashFoundation/zebra/releases) will have an automatic version bump. This version is based on [the labels on the PRs in the release](https://github.com/ZcashFoundation/zebra/blob/main/.github/release-drafter.yml).
 
 Check that the automatic `zebrad` version increment is correct:
-1. If we're releasing a mainnet network upgrade, increment the `major` version of all Zebra crates
-2. If we're not releasing a mainnet network upgrade, check for features, major changes, deprecations, and removals. If this release has any, it is a `minor` release
-
-If we're not doing a `major` release, you need to check which crates have changed:
-1. Go to the zebra GitHub code page: https://github.com/ZcashFoundation/zebra
-2. Check if the last commit to each crate is a Zebra version bump. If it is a version bump, the crate has not changed since the last release.
-   
-Once you know which crates have changed:
-- [ ] Increment the crates that have new commits since the last version update
-- [ ] Increment any crates that depend on crates that have changed
-- [ ] Keep a list of the crates that haven't been incremented, to include in the PR
-
-### How to Increment Versions
 
-Zebra follows [semantic versioning](https://semver.org).
+If we're releasing a mainnet network upgrade, it is a `major` release:
+1. Increment the `major` version of _*all*_ the Zebra crates.
+2. Increment the `patch` version of the tower crates.
 
-Semantic versions look like: MAJOR`.`MINOR`.`PATCH[`-`TAG`.`PRE-RELEASE]
+If we're not releasing a mainnet network upgrade, check for features, major changes, deprecations, and removals. If this release has any, it is a `minor` release:
+1. Increment the `minor` version of `zebrad`.
+2. Increment the `pre-release` version of the other crates.
+3. Increment the `patch` version of the tower crates.
 
-### Reviewing Version Bumps
+Otherwise, it is a `patch` release:
+1. Increment the `patch` version of `zebrad`.
+2. Increment the `pre-release` version of the other crates.
+3. Increment the `patch` version of the tower crates.
 
-Check for missed changes by going to:
-`https://github.com/ZcashFoundation/zebra/tree/<commit-hash>/`
-Where `<commit-hash>` is the hash of the last commit in the version bump PR.
+Zebra's Rust API is not stable or supported, so we keep all the crates on the same beta `pre-release` version.
```

Comment:
I'm not sure I fully understand how the versioning is going to look under this scenario and it makes me think it's going to be quite confusing to keep most crates on pre-release versions whilst `zebrad` is "stable"(?)

Can we keep this the way it was while we make a decision? Or has the decision been made already and I just missed it? :)

Reply:
We already decided that we aren't going to promise any support for the Zebra Rust APIs. That involves a lot of extra work, including checking all the public APIs and documentation, and automating the detection of breaking API changes. (We haven't done this work, and it isn't scheduled any time soon.)

To show that we aren't going to support the Rust APIs, we need to keep the crates on a `beta` pre-release version. This is pretty standard, it is what ECC did while they were developing Orchard and Halo 2.

If this gets too confusing for developers, we can automate the version bumps using a script or a tool.

If it gets too confusing for users, we can tell them "please ignore the Zebra crate versions, they aren't supported".